news/_posts/performance/2020-12-07-performance-and-prestashop.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title:  "Performance and PrestaShop"
+subtitle: "Sorry, Goose, but it's time to buzz the tower."
+date:   2020-12-04 9:00:00
+authors: [ Claude-ArnaudPerrot ]
+icon: icon-airplane
+tags:
+ - performance
+ - whitepaper
+---
+
+Let's start discussing performance and tuning with PrestaShop! 
+
+# Performance (& PrestaShop)
+
+You may have heard of it, but we'll soon release a White Paper, which is all about PrestaShop's performance, how to benchmark it and how it behaves under loads. And what to do to keep it responding under 300ms.
+
+Still, there are a number of matters that won't be covered in this White Paper - hopefully the first in a series, fingers crossed - that we'd love to shine a light on.
+
+Because, funny thing is, we often talk about performance as something we all understand and it is mostly true: we share the same expectations. 
```

Comment:
```suggestion
Because, funny thing is, we often talk about performance as something we all understand and it is mostly true: we share the same expectation. 
```

Reply:
Do you only have one expectation ? ;)